### PR TITLE
auth-utils: add support in the native image

### PR DESCRIPTION
**Pull request by ilappe**
## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: https://jira-hq.paloaltonetworks.local/browse/CIAC-861

## Related PRs
Related: https://github.com/demisto/dockerfiles/pull/19547

## Description
after the new image `auth-utils` will be merged in this [PR](https://github.com/demisto/dockerfiles/pull/19547) 
this PR will update the native image to support the new `auth-utils` image

### CODE REVIEW

**Review by JudahSchwartz — COMMENTED** [review]
@samuelFain @GuyAfik looks like a hard lock. We good with this? 
Otherwise looks good to me

File: docker/py3-native/pyproject.toml
Comment (JudahSchwartz) [inline docker/py3-native/pyproject.toml:72]: would lock to > version 2. Better to make sure in case something else gets added that it wont suddenly get switched

File: docker/py3-native/Dockerfile
Comment (GuyAfik) [inline docker/py3-native/Dockerfile:64]: please add to README.md that auth-utils is also supported

File: docker/py3-native/verify.py
Comment (GuyAfik) [inline docker/py3-native/verify.py:28]: do it in the auth-utils image verify.py

**Review by GuyAfik — COMMENTED** [review]
@ilappe great work!

1. You can add the validations  in the `verify.py` for the `auth-utils` in the `verify.py` of auth-utils image and then just add a symlink to it.
2. can we avoid locking the entire py3-native? it contains tons of packages which are getting updated, is it possible to update the dependancies only required for the new packages?

**Review by GuyAfik — APPROVED** [review]
looks good to me, please wait for @JudahSchwartz 's approval

**Comment by JudahSchwartz** [comment]
just left is my comment regarding pydantic, otherwise lgtm

**Pull request merged**
